src/helics/shared_api_library/helicsCallbacks.h

```diff
@@ -105,11 +105,13 @@ HELICS_EXPORT void helicsFilterSetCustomCallback(HelicsFilter filter,
  * to specific queries with answers specific to a federate.
  *
  * @param fed The federate to set the callback for.
- * @param queryAnswer A callback with signature const char *(const char *query, int querySize,int *answerSize, void *userdata);
+ * @param queryAnswer A callback with signature const char *(const char *query, int querySize, HelicsQueryBuffer buffer, void *userdata);
  *                 The function arguments are the query string requesting an answer along with its size, the string is not guaranteed to be
```

Comment:
Description doesn't include the userdata argument.

```suggestion
 *                 The function arguments include the query string requesting an answer along with its size; the string is not guaranteed to be
```